cfgov/cfgov/settings/base.py

```diff
@@ -550,11 +550,14 @@
                 '*.googletagmanager.com',
                 '*.google-analytics.com',
                 'www.youtube.com',
-                '*.doubleclick.net')
+                '*.doubleclick.net',
```

Comment:
Is this because GTM checks for add impressions/clicks? Is there a way to disable doubeclick from the tag manager side?